Dangerfile

```diff
@@ -9,6 +9,11 @@ unless github.branch_for_base == 'release'
   warn("Please assign someone to your PR", sticky: false) unless has_assignee
 end
 
+# Warn if adding new files that aren't .ts or .tsx
+if git.added_files.include? ".js"
+  warn('All new files should be `.ts` or `.tsx`. JavaScript has been depreciated.')
+end
```

Comment:
@orta - this _seems_ right -- any clue why danger didn't warn due to added file at the bottom?

Reply:
Think it needs to be a glob

Reply:
Ah, the `added_files` is an array, so you're asking if a file named `.js` has been added. You'll have to do a `select` or something on the array to check if it includes any file that ends in `.js`.

Reply:
we ended up adding a danger js dangerfile + peril to this repo instead, as Danger ruby was removed when moving to hokusai